### ISSUE TITLE
colors for animation

### ISSUE THREAD

**Issue by amjad489** [bprinty/animation#7]
this is a very usefull tool and i use it everywhere. it would be good if we have colors in animation.

**Comment by felixgao-0** [comment]
agree

**Comment by bprinty** [comment]
Added a `color=` argument to the decorator. Functionality will be available in version `0.0.7`

**Comment by bprinty** [comment]
For example:
```
    @animation.wait('bar', color='blue')
    def long_running_function():
        import time
        time.sleep(5)
        return
```
